generators/tracegen/src/main/scala/Configs.scala

```diff
@@ -10,7 +10,7 @@ import freechips.rocketchip.rocket.DCacheParams
 import freechips.rocketchip.tile.{MaxHartIdBits, XLen}
 import scala.math.{max, min}
 
-class WithTraceGen(params: Seq[DCacheParams], nReqs: Int = 8192)
```

Comment:
This really should be in a separate config mixin.

Reply:
It might be useful to tracegen a boom tile and a rocket tile together. Having 1 mixin for both types enables this.

Reply:
You can still use both with separate config mixins.

Though you will have to change the part where you instantiate them to check whether or not the parameter is defined or not.